tests/test_runtime.py

```diff
@@ -551,23 +550,29 @@ def test_too_many_param(self, token, tol, shots):
 
         def vqe_circuit(params):
             qml.RX(params[0], wires=0)
-            qml.RX(params[1], wires=0)
+            qml.RY(params[1], wires=0)
```

Comment:
How come this was changed?

Reply:
It makes more sense to have two different operations here, anyway we are not interested in the actual result.